third_party/move/move-core/types/src/value.rs

```diff
@@ -691,7 +704,9 @@ impl serde::Serialize for MoveValue {
             MoveValue::U128(i) => serializer.serialize_u128(*i),
             MoveValue::U256(i) => i.serialize(serializer),
             MoveValue::Address(a) => a.serialize(serializer),
-            MoveValue::Signer(a) => a.serialize(serializer),
+            MoveValue::Signer(a) => {
```

Comment:
MoveValue::Signer doesn't change?

Reply:
We never need to construct a permissioned signer on the rust side so I didn't add it. I was concerned if changing the value can bring more downstream impact to the indexer and annotation logic.